src/testsets.jl

```diff
@@ -15,22 +33,41 @@ flatted upon `finish`. See `bin/reporttests.jl` for an example of this use.
 and this is not recommended or supported.
 
 A `ReportingTestSet` has the `description` and `results` fields as per a
-`DefaultTestSet`, and has an additional `properties` field which is used
-to record properties to be inserted into the report.
+`DefaultTestSet`, and has the following additional fields:
+
+- `properties`: a dictionary which is used to record properties to be
+    inserted into the report.
+- `start_time::DateTime`: the start date and time of the testing (UTC).
```

Comment:
Is it UTC or is it system time? `now()` returns system time.
As does `time()`

Reply:
`time()` gives unix time, and then `unix2datetime(time())` gives UTC. But with the change to using `Millisecond`s it will now be in local time so will update this and the docs.